packages/orchestration/src/types.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-use-before-define -- fine for types */
+// Ambients
+import '@agoric/zoe/exported.js';
+
+import type { Amount, Brand } from '@agoric/ertp/exported.js';
+import type {
+  CoinAmount,
+  CosmosChainInfo,
+  CosmosStakingFacet,
+  Denom,
+  IcaAccount,
+} from './cosmos-api.js';
+import type { EthChainInfo } from './ethereum-api.js';
+import type { ICQConnection } from './exos/icqConnectionKit.js';
+import type {
+  ChainAddress,
+  OrchestrationAccountI,
+  TransferMsg,
+} from './orchestration-api.js';
+
+export type * from './cosmos-api.js';
+export type * from './exos/chainAccountKit.js';
+export type * from './exos/icqConnectionKit.js';
+export type * from './orchestration-api.js';
+export type * from './service.js';
+export type * from './vat-orchestration.js';
+
+/**
+ * static declaration of known chain types will allow type support for
+ * additional chain-specific operations like `liquidStake`
+ */
+export type KnownChains = {
+  stride: {
+    info: CosmosChainInfo;
+    methods: {
+      liquidStake: (amount: AmountArg) => Promise<void>;
+      // ??? do we prefer to merge capabilities into a single object? If we do
+      // that, then the chain-native methods (e.g. delegate) are hard to
+      // distinguish from generic. Alternately, there could be a method to get a
+      // generic facade on the chain-native methods.
+      getIcaAcccount: () => IcaAccount;
+      getIcqConnection: () => ICQConnection;
+      getStakingFacet: () => CosmosStakingFacet;
+    };
+  };
+  cosmos: { info: CosmosChainInfo; methods: {} };
+  agoric: {
+    info: Omit<CosmosChainInfo, 'ibcConnectionInfo'>;
```

Comment:
Omit is magic. What's this mean?

Reply:
How is it magic?

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

Reply:
Seems more clear to construct types additively rather than with subtraction. So what is the name of the type that doesn't include those operations?

Reply:
fair. Agoric is one special case among Cosmos chains so I don't think it needs its own type alias, but I'll change it to Pick instead of Omit

Reply:
use info so it looks like a localhost connection. Eventually it may be replaced with that implementation.